<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateConfiguration</class>
 <widget class="QMainWindow" name="CreateConfiguration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Configuration</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="generatePushButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>110</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Config</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="operationalGroupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>261</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Operational</string>
    </property>
    <widget class="QLineEdit" name="operationalLineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="activityGenerateGroupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>511</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Activity</string>
    </property>
    <widget class="QScrollArea" name="activityGenerateScrollArea">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>511</width>
       <height>80</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>509</width>
        <height>78</height>
       </rect>
      </property>
      <widget class="QPlainTextEdit" name="activityGeneratePlainTextEdit">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>511</width>
         <height>79</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="spareGroupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>261</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Spare</string>
    </property>
    <widget class="QLineEdit" name="spareLineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="selectopdevicePushButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>80</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selectsparedevicePushButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>160</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
